File: SERVIDOR SDR_SCR/lib/domain/boundary/output/get_attendance_control_list_presenter.py
from abc import ABC, abstractmethod
import asyncio
from lib.domain.dto.response.get_attendance_control_list_response import GetAttendanceControlListResponse

class GetAttendanceControlListPresenter(ABC):
    @abstractmethod
    async def displayAttendanceControlList(self, response: GetAttendanceControlListResponse): pass

File: SERVIDOR CENTRAL/lib/domain/boundary/output/get_attendance_control_list_presenter.go
package output

import "gitlab.com/jorgechavezrnd/servidorseminariogo2/lib/domain/dto/response"

// GetAttendanceControlListPresenter ...
type GetAttendanceControlListPresenter interface {
	DisplayAttendanceControlList(response response.GetAttendanceControlListResponse)
}
